chromadb/test/distributed/test_sanity.py

```diff
@@ -96,3 +96,43 @@ def test_add_include_all_with_compaction_delay(client: ClientAPI) -> None:
         10,
         query_embeddings=[random_query_1, random_query_2],
     )
+
+
+@skip_if_not_cluster()
```

Comment:
why are these cluster only and in test sanity? shouldn't we just do test/api/collection or similar?